Makefile

```diff
@@ -167,16 +167,16 @@ debug-shared: shared
 static: lib/libsass.a
 shared: $(LIBRARIES)
 
-lib/libsass.a: $(COBJECTS) $(OBJECTS)
+lib:
 	$(MKDIR) lib
```

Comment:
> The problem is that mkdir returns a failed exit state if it tries to re-create an already existing directory

Do we care if the directory already exists? I assume if the directory exists we can just continue on as normal correct?

> And there doesn't seem to be an easy portable way to test for existing directories!

Assume the above is correct I think you can do the following `@$(MKDIR) lib || true`. This will just will suppress the error that the direct already (and any other errors unfortunately).